Data/Sys/InfoDisplay/RMCE01.ini

```diff
@@ -28,21 +27,23 @@ End Line
 
 Line5="===== Misc =====
 Base Vel: %f4b | Max: %f4b
-MT Charge: %d2b (%d2b)
-SSMT Charge: %d2b
-Airtime: %d2b
-MT Boost: %d2b
-Trick Boost: %d2b
-Mushroom Boost:  %d2b"
+MT Charge: %d2b (%d2b) | SSMT Charge: %d2b
+MT: %d2b | Trick: %d2b | Mushroom:  %d2b
+Trick CD: %d2b | Wheelie CD: %d2b | Wheelie: %d4b
+Airtime: %d2b | OOB: %d2b"
 Arg1=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x20;
 Arg2=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x18;
 Arg3=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0xFE;
 Arg4=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x100;
 Arg5=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x14C;
-Arg6=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x21A;
-Arg7=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x10C;
-Arg8=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x114;
-Arg9=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x110;
+Arg6=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x10C;
+Arg7=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x114;
+Arg8=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x110;
+Arg9=0x9BD110 + 0xC + 0x10 + 0x0 + 0x10 + 0x10 + 0x21A;
```

Comment:
Arg9 is airtime, yet it writes to Trick CD.
Arg10 is trick cooldown, yet it writes to Wheelie CD.
Arg11 is wheelie CD, yet it writes to Wheelie.
Arg12 is wheelie frames, yet it writes to Airtime.

Airtime is also 4 bytes and resides at 0x218, not 0x21A.

Reply:
Addressed in https://github.com/TASLabz/dolphin-lua-core/pull/18/commits/66d3ee2845ae139a01a37672e1eb9ff2bf26bb4a.